@here/harp-examples/src/geo-to-screen.ts

```diff
@@ -0,0 +1,286 @@
+            heading: -80
+        });
+        map.renderLabels = false;
+
+        CopyrightElementHandler.install("copyrightNotice", map);
+
+        const mapControls = new MapControls(map);
+        mapControls.maxTiltAngle = 180;
+
+        const ui = new MapControlsUI(mapControls, { zoomLevel: "input" });
+        canvas.parentElement!.appendChild(ui.domElement);
+
+        map.resize(window.innerWidth, window.innerHeight);
+
+        window.addEventListener("resize", () => {
+            map.resize(window.innerWidth, window.innerHeight);
+        });
+
+        addVectorTileDataSource(map);
+
+        return map;
+    }
+
+    function addGuiElements() {
+        const gui = new GUI({ width: 300 });
+        gui.add(geoPosition, "lat")
+            .step(0.0001)
+            .onChange(updateAnchors);
+        gui.add(geoPosition, "lng")
+            .step(0.0001)
+            .onChange(updateAnchors);
+        gui.add(geoPosition, "alt")
+            .step(0.0001)
+            .onChange(updateAnchors);
+        gui.add(mapView, "tileWrappingEnabled").onChange((value: boolean) => {
+            mapView.tileWrappingEnabled = value;
+        });
+        const options = {
+            projection: "mercatorProjection"
+        };
+        gui.add(options, "projection", ["sphereProjection", "mercatorProjection"]).onChange(
+            (value: string) => {
+                mapView.projection =
+                    value === "sphereProjection" ? sphereProjection : mercatorProjection;
+            }
+        );
+        return gui;
+    }
+
+    function addVectorTileDataSource(map: MapView) {
+        const omvDataSource = new VectorTileDataSource({
+            baseUrl: "https://vector.hereapi.com/v2/vectortiles/base/mc",
+            authenticationCode: apikey
+        });
+
+        map.addDataSource(omvDataSource);
+
+        return map;
+    }
+
+    function updateAnchors() {
+        gui.updateDisplay();
+        const screenpos = mapView.getScreenPosition({
+            latitude: geoPosition.lat,
+            longitude: geoPosition.lng,
+            altitude: geoPosition.alt
+        });
+        if (screenAnchor !== undefined && screenpos !== undefined) {
+            screenAnchor.style.left = screenpos.x.toString() + "px";
+            screenAnchor.style.top = screenpos.y.toString() + "px";
+            screenAnchor.style.opacity = "0.8";
+            screenAnchorValues.innerHTML =
+                "x: " + Math.round(screenpos.x) + " y: " + Math.round(screenpos.y);
+        } else {
+            screenAnchor.style.opacity = "0.1";
+            screenAnchorValues.innerHTML = "x: undefined , y: undefined";
+        }
+        if (mapAnchor !== undefined && mapAnchor.geoPosition !== undefined) {
+            const scaleFactor = mapView.targetDistance / 10000;
+            if (mapAnchor.scale.x !== scaleFactor) {
+                mapAnchor.scale.set(scaleFactor, scaleFactor, scaleFactor);
+                mapView.requestUpdateIfNeeded();
```

Comment:
`requestUpdateIfNeeded` is deprecated. Just use `update`
It seems we lost the deprecation warnings when we switching from tslint to eslint. Can you create a ticket for that please.

Reply:
done